### PR TITLE
Update hotspots list styling

### PR DESCRIPTION
**Pull request by matthewcarlreetz**
This mostly matches figma, but there are some design pieces that still need to be finalized. Decided to merge this progress and will make another pass at it later.

### CODE REVIEW

File: src/features/hotspots/root/HotspotsTicker.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  )
+  const blockTime = useSelector(
+    (state: RootState) => state.heliumData.blockTime,
+  )
+
+  useEffect(() => {
+    dispatch(fetchCurrentOraclePrice())
+    dispatch(fetchPredictedOraclePrice())
+    dispatch(fetchStats())
+  }, [dispatch])
```
Comment (tyler-whitman) [inline src/features/hotspots/root/HotspotsTicker.tsx:38]: Do we want to put this on a 5 min update interval? That is what the rest of the page is using for rewards info. If you check in `WelcomeOverview` there is a date that gets updated every 5 min, you could just pass it down and add it to the dependencies so this refreshes too?

Comment (tyler-whitman) [reply src/features/hotspots/root/HotspotsTicker.tsx:38]: Or we could maybe make it like 15 or 30 minutes and do it within this component?

**Review by tyler-whitman — APPROVED** [review]

File: src/features/hotspots/root/HotspotsTicker.tsx (hunk 1 above)
Comment (matthewcarlreetz) [reply src/features/hotspots/root/HotspotsTicker.tsx:38]: Maybe I should do it every time it focuses?

Comment (tyler-whitman) [reply src/features/hotspots/root/HotspotsTicker.tsx:38]: Hmm just dont want to hammer the API. I think every 5 min is enough

Comment (matthewcarlreetz) [reply src/features/hotspots/root/HotspotsTicker.tsx:38]: What if I combine them both? On focus, if it's been more than 5 minutes.

Comment (tyler-whitman) [reply src/features/hotspots/root/HotspotsTicker.tsx:38]: That sounds good

**Pull request merged**